configure

```diff
@@ -8143,7 +8144,7 @@ fi
     if test "x$compiler_brand_detected" = "xno"
 then :
 
-          is_intel_icc="`($CXX -V 2>&1) | grep 'Intel(R)' | grep 'Compiler' | grep -v 'oneAPI'`"
+          is_intel_icc="`($CXX -V 2>&1) | grep 'Intel(R)' | grep 'Compiler'`"
```

Comment:
Are you working with an up-to-date version of the autoconf-submodule? Your diff is removing this grep statement that refers to `oneAPI` that was recently added; I don't think we want to do that.

Reply:
should be it's all from a fresh clone

Reply:
nope nevermind it was not